SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -61,22 +83,14 @@ USpatialGDKEditorSettings::USpatialGDKEditorSettings(const FObjectInitializer& O
 	SpatialOSSnapshotToLoad = GetSpatialOSSnapshotToLoad();
 }
 
-const FString& USpatialGDKEditorSettings::GetSpatialOSRuntimeVersionForLocal() const
-{
-	if (bUseGDKPinnedRuntimeVersion || LocalRuntimeVersion.IsEmpty())
-	{
-		return SpatialGDKServicesConstants::SpatialOSRuntimePinnedVersion;
-	}
-	return LocalRuntimeVersion;
-}
-
-const FString& USpatialGDKEditorSettings::GetSpatialOSRuntimeVersionForCloud() const
+FRuntimeVariantVersion& USpatialGDKEditorSettings::GetRuntimeVariantVersion(ESpatialOSRuntimeVariant::Type Variant)
 {
-	if (bUseGDKPinnedRuntimeVersion || CloudRuntimeVersion.IsEmpty())
+	if (Variant == ESpatialOSRuntimeVariant::Distributed)
 	{
-		return SpatialGDKServicesConstants::SpatialOSRuntimePinnedVersion;
+		return DistributedRuntimeVersion;
 	}
-	return CloudRuntimeVersion;
+	check(Variant == ESpatialOSRuntimeVariant::SingleNode);
```

Comment:
What's with the check?

Reply:
Just to make say we do not have any other values than the two known ones.